mrbgems/mruby-compiler/core/parse.y

```diff
@@ -3452,7 +3452,7 @@ backref_error(parser_state *p, node *n)
   c = (int)(intptr_t)n->car;
 
   if (c == NODE_NTH_REF) {
-    yyerror_i(p, "can't set variable $%" MRB_PRId, (mrb_int)(intptr_t)n->cdr);
+    yyerror_i(p, "can't set variable $%" MRB_PRId, (int)(mrb_int)(intptr_t)n->cdr);
```

Comment:
This triple cast appears weird. I think `(int)(intptr_t)` should work.

Reply:
Fixed.